vignettes/TutorialDaySegmentAnalyses.Rmd

```diff
@@ -95,6 +98,15 @@ ID | date | A | B | C
 - Dates with no activity log data can be skipped, no need to have a column with the date followed by a column with the next date.
 - The end time of one activity is assumed to be the start time of the next activity. We currently do not facilitate overlapping time segments.
 
+# Implementation in GGIR
+
+Both approaches are implemented in GGIR part 2 and part 5. Therefore the specific
+output variables that are calculated both in part 2 and 5 are available per day,
+per person, and per segment of the day based on the argument qwindow.
+
+For more information about the output variables calculated in each part
+of the pipeline, see the main [GGIR vignette](https://CRAN.R-project.org/package=GGIR/vignettes/GGIR.html).
+
```

Comment:
I imagine most users will not be interested in this for both part 2 and 5, so it may be good to clarify how user can only run qwindow in part 2 and not in part 5, or only in part 5 and not in part 2. For example: Explain that this is not facilitate at the moment, but that a work around would be to first run GGIR with mode = 1:2 and qwindow with values, and then with mode = 3:5 while setting qwindow to NULL or visa versa.